README.md

```diff
@@ -1,2 +1,137 @@
-# Budget
-A budget app build with rails, this is the capstone project for the Rails module, part of Microverse.org Fullstack program
+<a name="readme-top"></a>
+<div align="center">
+  <img src="./budget.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Budget on Rails</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget ](#-budget-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget <a name="budget"></a>
+
+**Budget** is a budgeting app build with rails, this is the capstone project for the Rails module, part of Microverse.org Fullstack program
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
```

Comment:
- [ ] You're expected to record a video with a maximum length of 5 minutes. The content of the video should include:

- a description of the project.
- a demo of the project features.
- You should also highlight some interesting piece of code or something you built that you are very proud of.
For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.

Reply:
I update it on the same video
https://www.loom.com/share/174976d4be1b498582735f140add107c